src/transformers/models/whisper/processing_whisper.py

```diff
@@ -91,3 +96,22 @@ def decode(self, *args, **kwargs):
         the docstring of this method for more information.
         """
         return self.tokenizer.decode(*args, **kwargs)
+
+    def get_prompt_ids(self, text: str):
+        """Converts prompt text to IDs that can be passed to [`~WhisperForConditionalGeneration.generate`]."""
+        _verify_no_special_tokens(text, self.tokenizer.all_special_tokens)
+        prompt_token = "<|startofprev|>"
```

Comment:
This can be defined nicely in the `tokenizer` file once we move the method there!